src/applications/vaos/appointment-list/components/AppointmentsPageV2/AppointmentListItem.jsx

```diff
@@ -13,7 +13,6 @@ import {
   VIDEO_TYPES,
   SPACE_BAR,
 } from '../../../utils/constants';
-import { updateBreadcrumb } from '../../redux/actions';
 import { selectFeatureStatusImprovement } from '../../../redux/selectors';
 
 function VideoAppointmentDescription({ appointment }) {
```

Comment:
Small but I would add PropTypes. We've been trying to get rid of the easy eslint warning when we touch files.

Reply:
Will do, was wondering about that.